buildscripts/kokoro/android-interop.sh

```diff
@@ -15,6 +15,9 @@ export LDFLAGS=-L/tmp/protobuf/lib
 export CXXFLAGS=-I/tmp/protobuf/include
 export LD_LIBRARY_PATH=/tmp/protobuf/lib
 export OS_NAME=$(uname)
+export ANDROID_SDK_HOME=/opt/android-sdk/current
```

Comment:
ANDROID_HOME is already defined with this value.

Reply:
Is `ANDROID_HOME` already defined for both `android` and `android-interop-test` in kokoru env?

Reply:
Kokoro defines ANDROID_HOME. I think the android gradle plugin itself requires ANDROID_HOME to be set.